WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -988,11 +1002,11 @@ class AbstractPostListViewController : UIViewController, WPContentSyncHelperDele
         }
     }
 
-    func keyForCurrentListStatusFilter() -> String {
+    class func keyForCurrentListStatusFilter() -> String {
         fatalError("You should implement this method in the subclass")
     }
 
-    func currentFilterIndex() -> Int {
+    class func currentFilterIndex() -> Int {
```

Comment:
> Unify the instance and class availablePostListFilters() methods. Having two wouldn't be good.

Let me rephrase that: it's not so much about having the same method twice, as much as the class method right now feeling like a quick patch since we can't call the instance one.